exonum/src/blockchain/service.rs

```diff
@@ -400,19 +402,37 @@ impl SharedNodeState {
 
         lock.peers_info.clear();
         lock.majority_count = state.majority_count();
+        lock.is_validator = state.is_validator();
+        lock.validators = state.validators().to_vec();
 
-        for (p, c) in state.peers().iter() {
+        for (p, c) in state.peers() {
             lock.peers_info.insert(c.addr(), *p);
         }
     }
 
-    /// Returns the majority count from the current "State"
-    /// of a blockchain node.
-    pub fn majority_count(&self) -> usize {
-        self.state
-            .read()
-            .expect("Expected read lock.")
-            .majority_count
+    /// Returns a boolean value which indicates whether the consensus is
```

Comment:
`Whether the consensus is...`
A bit of intrigue? :)

Reply:
)) sorry "achieved" missed